tests/beamlines/unit_tests/test_device_helpers.py

```diff
@@ -0,0 +1,23 @@
+from random import randint
+
+from dodal.beamlines._device_helpers import numbered_slits
+
+
+def test_slit_field_naming():
+    num = randint(1, 20)
+    prefix = f"-AL-SLITS-{num:02}:"
+    slits = numbered_slits(
+        num,
+        False,
+        True,
```

Comment:
I think it would be clearer to pass these as kwargs.

Reply:
Do you mean 

```
    slits = numbered_slits(
        num,
        wait_for_connection=False,
        fake_with_ophyd_sim=True,
    }
```

?

Reply:
Yes exactly. Certainly for the two booleans, I'd probably do it for `num` as well personally but don't mind so much about that one

Reply:
ok